app/javascripts/graphiql.jsx

```diff
@@ -6,6 +6,10 @@ import {AppConfig} from './config';
 import './css/graphiql.css'
 import 'graphiql/graphiql.css'
 
-const config = new AppConfig("graphiql")
-
-ReactDOM.render(<GraphiQLTool config={config} />, document.getElementById('graphiql-tool'));
\ No newline at end of file
+module.exports = {
+  bootstrap: function (bootstrapOptions = {}) {
+    this.bootstrapOptions = bootstrapOptions;
+    const config = new AppConfig("graphiql", bootstrapOptions);
+    ReactDOM.render(<GraphiQLTool config={config} />, document.getElementById('graphiql-tool'));
```

Comment:
I though about it but decided to not do it for now with the rational that in 99.99%, the tool is on its own on the page and therefore it's not really problematic to force the user to use a given element ID.

Now, it's really trivial to do and I can include it.